common/persistence/versionhistory/version_histories.go

```diff
@@ -192,3 +192,12 @@ func SetCurrentVersionHistoryIndex(h *historyspb.VersionHistories, currentVersio
 func GetCurrentVersionHistory(h *historyspb.VersionHistories) (*historyspb.VersionHistory, error) {
 	return GetVersionHistory(h, h.GetCurrentVersionHistoryIndex())
 }
+
+// IsCurrentVersionHistoryEmpty checks if the current VersionHistory is empty.
+func IsCurrentVersionHistoryEmpty(h *historyspb.VersionHistories) (bool, error) {
+	currentVersionHistory, err := GetCurrentVersionHistory(h)
```

Comment:
return true if version histories is empty?

Reply:
I reverted my change to all empty version histories. 
I think version histories should never be empty. Even if mutable state have no events at all, the version histories can have one version history when empty items.